src/framework/theme/component/application/applicationProvider.component.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { ReactNode } from 'react';
+import {
+  ThemeMappingType,
+  ThemeMapType,
+} from 'eva/packages/common';
+import { MappingProviderProps } from '../mapping';
+import { ThemeProviderProps } from '../theme';
+import { ThemeType } from '../../type';
+import { ModalPanel } from '../modal';
+import { StyleProvider } from '../style';
+
+interface ApplicationProviderProps {
+  children: JSX.Element | ReactNode | null;
```

Comment:
`ReactNode` would be enough for this
I even suggest not support passing nulls and other unexpectable parameters as children
You should strict this to `React.ReactElement | React.ReactElement[]`